frontend/src/component/releases/ReleasePlanTemplate/ReleasePlanTemplateAddStrategyForm.tsx

```diff
@@ -135,8 +136,9 @@ export const ReleasePlanTemplateAddStrategyForm = ({
 }: IReleasePlanTemplateAddStrategyFormProps) => {
     const [addStrategy, setAddStrategy] = useState(strategy);
```

Comment:
Technically not an "add" strategy anymore, also used for the edited strategy. So should update the name. currentStrategy?